src/Microsoft.OpenApi/Validations/Rules/OpenApiRuleAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.OpenApi.Validations.Rules
+{
+    /// <summary>
+    /// The Validator attribute.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
```

Comment:
>Class [](start = 37, length = 5)

Why class attribute vs field/property attribute?

Reply:
It's easy to target "Rule" Class. 

If it's target on the Rule property, we should go through all the properties in all the types, it's time consuming.